packages/nimble-components/src/table/specs/table-column-sort-hld.md

```diff
@@ -34,15 +25,24 @@ The column definition will also be responsible for specifying the appropriate so
 
 For performance reasons, the table will not support custom sort functions on a column. Additionally, a column can only be sorted based on a single field within the table's data. These requirements guard against performance degradations from inefficient custom sort functions.
 
+Interactive sorting will be controlled by a new `sortMode` property on the `Table` (of type `TableInteractiveSortMode`):
```

Comment:
Do we need to disambiguate this property from what the programmatic API allows by calling this `interactiveSortMode`?

Reply:
Added a Open Issue for whether we want to include an `interactive` prefix here, and potentially on `sortingDisabled` too.

Reply:
Based on the other conversation thread, removed `sortMode` since we currently don't have a use case that requires it.